administration/urls.py

```diff
@@ -1,9 +1,12 @@
-from django.urls import path
-from .views import UsersListView, UserDetailView
+from django.urls import path, re_path, include
+
+from administration.views import AdminProfileList, AdminProfileDetail
 
 app_name = "administration"
 
 urlpatterns = [
-    path('users/', UsersListView.as_view(), name='users-list'),
-    path('users/<pk>/', UserDetailView.as_view(), name='user'),
+    path("auth/", include('djoser.urls')),
+    re_path(r"^auth/", include('djoser.urls.authtoken')),
```

Comment:
Is this correct? Do we need `djoser` for authentication, we have strong reasons to consider it better/simpler than Django's built-in users authentication with Django sessions and cookies?

Reply:
We also discussed this and decided to keep the authentication that has already been implemented.
Again, we need the requirements for the admin panel so that we can do everything correctly.